Assets/Monsters/MonsterAI.cs

```diff
@@ -28,13 +28,16 @@ private void Update()
     {
         if (target == null)
         {
-            if(gameObject.tag == ("Team A"))
+            if (gameObject.tag == ("Team A"))
```

Comment:
we need to eventually add constants for "Team A" and "Team B" so we don't accidentally misspell it